mdnDocsOverrides.json

```
@@ -350,19 +370,28 @@
   "capture-stream-audio-video": [
     "Web/API/HTMLMediaElement/captureStream"
   ],
-  "capture-stream-canvas": [],
+  "capture-stream-canvas": [
+    "Web/API/CanvasCaptureMediaStreamTrack"
+  ],
```

Makes more sense to me if we link directly to the method that's mentioned in the feature:

```suggestion
  "capture-stream-canvas": [
    "Web/API/HTMLCanvasElement/captureStream"
  ],
```